include/wx/localedefs.h

```diff
@@ -89,6 +89,19 @@ enum wxLocaleInfo
     // the character used as decimal point (for wxLOCALE_CAT_NUMBER or MONEY)
     wxLOCALE_DECIMAL_POINT,
 
+    // check whether locale uses the metric system for measurements
+    // returns "Yes", "No", or "Unknown"
```

Comment:
I'd probably rather use "1", "0" and "" instead of these strings.

In any case, let's define `wxLOCALE_TRUE`, `_FALSE` and `_UNKNOWN` constants for them instead of using the literal strings.

Alternatively, perhaps it would be better to have a `wxLocaleMeasurement` enum and a separate function `GetMeasurementSystem()` returning it? This would be much cleaner IMO.